.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": false,
+  "trailingComma": "all"
+}
```

Comment:
I have prettier as an autoformatter, I configured it so it won't change the project's code style.